README.md

```diff
@@ -44,6 +44,7 @@ building with makepkg:
 Before you begin, make sure you have the `base-devel` package group installed.
 
 ```sh
+pacman -S git fakeroot binutils gcc make
```

Comment:
```suggestion
pacman -S git base-devel
```

Reply:
You can clean up build artifacts by using a multi-stage Dockerfile, however downloading packages that are never used goes against [Lean Flow](https://www.google.com/search?q=lean+flow) principles.  Your batch size and process time will not be as small as possible.  Determine that `git fakeroot binutils gcc make` are the smallest number of dependencies takes time too.

Reply:
If having the smallest subset of packages to build yay and _only_ yay is your goal, your argument is reasonable.

However, I would consider another goal, namely that _after_ installing yay, you want to install other AUR packages. And many of those have the implicit assumption that _all_ of base-devel is installed (and won't list them in their dependencies). Failure to instruct users to install base-devel completely will probably result in people coming here or elsewhere to complain about missing package dependencies.

Reply:
https://wiki.archlinux.org/index.php/Arch_User_Repository#Getting_started

`Ensure the base-devel package group is installed in full (pacman -S --needed base-devel).`

it's a "minimum requirement" for using the AUR. 

A more experienced user will know how to install with the bare minimum of dependencies. If you want as a compromise you may add a note that to build yay itself only x,y,z are used from base-devel

Reply:
> you may add a note that to build yay itself only x,y,z are used from base-devel

But please in an extra section, the installation instructions should be as brief and precise as possible, as not to confuse newcomers.